torch/csrc/lazy/core/tensor_impl.cpp

```diff
@@ -162,7 +189,8 @@ void LTCTensorImpl::setup_size_properties() {
 at::IntArrayRef LTCTensorImpl::sizes() const {
   // NOLINTNEXTLINE(cppcoreguidelines-pro-type-const-cast)
   const_cast<LTCTensorImpl*>(this)->setup_size_properties();
-  return c10::TensorImpl::sizes();
```

Comment:
Are we no longer using `c10::TensorImpl::sizes()`?

Reply:
`c10::TensorImpl::sizes()` would return symbolic sizes for LTC since this is what we are storing inside `sizes_and_strides_`

Reply:
Please see : 

https://github.com/pytorch/pytorch/blob/51618cb3bfbc801d23bbcdee5081090669b18762/torch/csrc/lazy/core/tensor_impl.cpp#L91-L101